platform/broadcom/saibcm-modules/systems/linux/kernel/modules/bcm-knet/bcm-knet.c

```diff
@@ -5279,7 +5279,7 @@ bkn_tx(struct sk_buff *skb, struct net_device *dev)
     }
 
     if (!netif_carrier_ok(dev)) {
-        DBG_WARN(("Tx drop: Invalid RCPU encapsulation\n"));
+        DBG_WARN(("Tx drop: Netif link is down.\n"));
```

Comment:
is there way to get the device name in the log?

Reply:
May be print `ifalias `or `name `field?

Reply:
**dev->name** looks ok. But as a general comment, most of the debugs in this file don't have device name printed, may be to suggest brcm to improve debugs.

Reply:
@prsunny and @judyjoseph Sure, will ask Broadcom to enhance debug messages. For now since this change same as what is in BRCM-SAI 4.2 will keep same so to avoid local changes.